cpp/src/aggregation/result_cache.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2021, NVIDIA CORPORATION.
```

Comment:
We usually only update copyright when something else in the file changes...  But I guess it doesn't hurt.